server/protocols/jmap/src/test/java/org/apache/james/jmap/send/PostDequeueDecoratorTest.java

```diff
@@ -230,4 +239,31 @@ public void doneShouldNotMoveMailFromOutboxToSentWhenInvalidMessageIdProvided()
         MessageResultIterator resultIterator = mailbox.getMessages(MessageRange.one(UID), FetchGroupImpl.FULL_CONTENT, mailboxSession);
         assertThat(resultIterator).hasSize(1);
     }
+
+    @Test
+    public void doneShouldCopyMailFromOutboxToSentOnlyOneTimeWhenSuccess() throws Exception {
```

Comment:
`doneShouldCopyMailFromOutboxToSentOnlyOneTimeWhenSuccess` -> `doneShouldCopyMailFromOutboxToSentOnSuccessAndOnlyOneTime`.
I think it's clearer but maybe that is me